gum/backend-dbghelp/gumsymbolutil-dbghelp.c

```diff
@@ -148,6 +148,28 @@ gum_find_functions_matching (const gchar * str)
   return matches;
 }
 
+guint64
+gum_load_module(const gchar * str)
```

Comment:
```suggestion
gum_load_symbols (const gchar * path)
```

As this sounds too similiar to `gum_module_load()`, we'll need a more contextual name.